src/frontend/komponenter/Fagsak/TilkjentYtelse/TilkjentYtelse.tsx

```diff
@@ -0,0 +1,93 @@
+import { IFagsak } from '../../../typer/fagsak';
+import { Ingress, Innholdstittel, Undertittel } from 'nav-frontend-typografi';
+import AlertStripe from 'nav-frontend-alertstriper';
+import * as React from 'react';
+import { hentAktivBehandlingPåFagsak } from '../../../utils/fagsak';
+import { useApp } from '../../../context/AppContext';
+import { Ressurs, RessursStatus } from '../../../typer/ressurs';
+import { AxiosError } from 'axios';
+import { Knapp } from 'nav-frontend-knapper';
+import { Behandlingstype } from '../../../typer/behandling';
+import { useHistory } from 'react-router';
+import { IOppsummeringBeregning } from '../../../typer/beregning';
+import { Oppsummeringsrad, OppsummeringsradHeader } from './Oppsummeringsrad';
+import { datoformat, formaterIsoDato } from '../../../utils/formatter';
+
+interface ITilkjentYtelseProps {
+    fagsak: IFagsak;
+}
+
+const TilkjentYtelse: React.FunctionComponent<ITilkjentYtelseProps> = ({ fagsak }) => {
+    const { axiosRequest } = useApp();
+    const history = useHistory();
+    const [oppsummeringBeregning, setOppsummeringBeregning] = React.useState<
+        IOppsummeringBeregning[]
+    >([]);
+    const [errorMessage, setErrorMessage] = React.useState<string | undefined>(undefined);
+    const aktivBehandling = hentAktivBehandlingPåFagsak(fagsak);
+    React.useEffect(() => {
+        axiosRequest<IOppsummeringBeregning[], void>({
+            method: 'GET',
+            url: `/familie-ba-sak/api/vedtak/oversikt/${aktivBehandling?.behandlingId}`,
+        })
+            .then((response: Ressurs<IOppsummeringBeregning[]>) => {
+                if (response.status === RessursStatus.SUKSESS) {
+                    setOppsummeringBeregning(response.data);
+                    setErrorMessage(undefined);
+                } else if (response.status === RessursStatus.FEILET) {
+                    setErrorMessage(response.melding);
+                } else {
+                    setErrorMessage('Ukjent feil, kunne ikke vise tilkjent ytelse.');
+                }
+            })
+            .catch((_error: AxiosError) => {
+                setErrorMessage('Ukjent feil, Kunne ikke generere forhåndsvisning.');
+            });
+    }, []);
+    const startdato = oppsummeringBeregning[0]
+        ? formaterIsoDato(oppsummeringBeregning[0].periodeFom, datoformat.DATO_FORLENGET)
+        : '';
+    return (
+        <div className="tilkjentytelse">
```

Comment:
Kan vi bruke steg-komponenten for å bevare likhet her? Den inkluderer forrige og neste knapper.

Reply:
👍

Reply:
Ser at `Skjemasteg` har `max-width: 40rem;` @henninghaakonsen 
Er det meningen at ingen skjermbilder skal være større enn dette? Det ser litt knapt ut for tabellen vi skal vise frem,

Reply:
Bare å forandre det, bare satt fra tidligere. Kan også vurdere å legge til en prop for width på skjemasteg :) Har ikke helt skjønt meg på bredde for systemet, så må ta en prat med camilla og gunn rundt det senere

Reply:
👍